corehq/apps/app_manager/models.py

```diff
@@ -315,6 +315,9 @@ class FormActions(DocumentSchema):
     case_preload = SchemaProperty(PreloadAction)
     referral_preload = SchemaProperty(PreloadAction)
 
+    update_usercase = SchemaProperty(UpdateCaseAction)
+    usercase_preload = SchemaProperty(PreloadAction)
```

Comment:
Would be nice to name these with the same schema e.g. usercase_preload, usercase_update.